### DISCUSSION TITLE
Welcome to the JokeAPI Discussion Board!

### DISCUSSION THREAD

**Discussion by Sv443** [Sv443/JokeAPI#204] [General]
## 👋 Welcome!
  We're using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you're wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

Please remain friendly and avoid using explicit language.  
To get started, you can comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vuk12** [comment]
Hi all, thank you for you great project, and sharing it with others. 
I have one question. I want to create new joks using POST call as described in  https://v2.jokeapi.dev/submit 
Example Payload: with postman, but I got  "This joke was formatted incorrectly." Could you please try the same, just to see is there any error, new parametars or something else. Thanks.

**Reply by Sv443** [reply]
Thanks for using JokeAPI!
Please make sure your request body follows the format described [here.](https://v2.jokeapi.dev/#submit-endpoint)
If that didn't fix your problem, please open a new discussion thread [here.](https://github.com/Sv443/JokeAPI/discussions/new)
